apps/website/src/content/docs/progressindicator.mdx

```diff
@@ -1,29 +1,90 @@
 ---
 title: Progress indicator
-description: Progress indicators express an unspecified wait time or display the length of a process.
+description: Progress indicators express the length of in-progress operations.
 thumbnail: ProgressIndicator
 ---
 
 <p>{frontmatter.description}</p>
 
-## Progress linear
+## Usage
 
-<Placeholder componentPageName='progresslinear--determinate' />
+There are two types of progress indicators which are linear and radial. Linear indicators show progress on a linear bar, while radial indicators display progress in a circular form.
+
+The default progress indicator is indeterminate. If a [value](#value) is passed, the indicator becomes determinate.
```

Comment:
This paragraph could probably be removed/rephrased to fit into the "Value" section.

Reply:
Hmmm, probably I should just split it and move the second sentence where value is mentioned to the "Value" section?

Reply:
How about we keep it in the section but move it *below* the demo and expand on it to clarify what "indeterminate" is. This segues nicely into the next section.

> By default, the progress indicators are considered to be "indeterminate". This is useful for short processes, where it isn't necessary (or possible) to indicate the completion percentage. For longer processes, a "determinate" progress indicator should be used.

Reply:
I have made the change.